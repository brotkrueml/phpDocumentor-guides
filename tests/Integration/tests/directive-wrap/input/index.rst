Directive tests
===============

..  wrap::
    :class: someClass

    Lorem Ipsum Dolor
