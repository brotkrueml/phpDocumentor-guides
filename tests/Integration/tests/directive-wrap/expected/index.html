<!DOCTYPE html>
<html lang="en">
<head>
    <title>Directive tests</title>

</head>
<body>
    <div class="section" id="directive-tests">
    <h1>Directive tests</h1>

                <div class="someclass"><p>Lorem Ipsum Dolor</p></div>

    </div>

</body>
</html>
